client/src/dashboard/Assets.vue

```diff
@@ -31,6 +31,9 @@ const body = document.getElementsByTagName("body")[0];
 const activeSelectionUrl = `url(${baseAdjust("/static/img/assetmanager/active_selection.png")})`;
 const emptySelectionUrl = `url(${baseAdjust("/static/img/assetmanager/empty_selection.png")})`;
 
+const currentRenameElement = ref<HTMLDivElement | null>(null);
+const titleRefs = ref<Record<AssetId, HTMLDivElement | null>>({});
```

Comment:
I would make this either a `reactive<Record<AssetId, HTMLDivElement>>` or a `ref<Map<AssetId, HTMLDivElement>>`.

Given that it's an object, using `reactive` will be just a tad bit more ergonomic as you don't have to do the `.value` in the `setTitleRef` function. On the other hand I also just like a `Map` when it's an actual mapping over an object in which case a `ref` is needed.

Feel free to choose what you prefer

Reply:
I went with reactive. It was nicer to not have to say `.value` everywhere.